_ml-commons-plugin/api.md

```diff
@@ -0,0 +1,968 @@
+---
+layout: default
+title: API
+has_children: false
+nav_order: 99
+---
+
+# ML Commons API 
+
+---
+
+#### Table of contents
+- TOC
+{:toc}
+
+
+---
+
+The Machine Learning (ML) commons API lets you create, train, and store machine learning algorithms both synchronously and asynchronously.  
+
+In order to train tasks through the API, three inputs are required. 
+
+- Algorithm name: Usually `FunctionaName`. This determines what algorithm the ML Engine runs.
```

Comment:
Must be one of `FunctionaName`. If algorithm name is not function name, will throw error.

Reply:
Seems this comment not addressed yet

Reply:
Done now.